op-conductor/conductor/service.go

```diff
@@ -683,18 +687,27 @@ func (oc *OpConductor) transferLeader() error {
 }
 
 func (oc *OpConductor) stopSequencer() error {
-	oc.log.Info("stopping sequencer", "server", oc.cons.ServerID(), "leader", oc.leader.Load(), "healthy", oc.healthy.Load(), "active", oc.seqActive.Load())
-
-	_, err := oc.ctrl.StopSequencer(context.Background())
-	if err != nil {
-		if strings.Contains(err.Error(), driver.ErrSequencerAlreadyStopped.Error()) {
-			oc.log.Warn("sequencer already stopped.", "err", err)
+	oc.log.Info(
+		"stopping sequencer",
+		"server", oc.cons.ServerID(),
+		"leader", oc.leader.Load(),
+		"healthy", oc.healthy.Load(),
+		"active", oc.seqActive.Load())
+
+	// Getting stuck stopping a sequencer can't be good. Is it okay to fail to stop a sequencer on
+	// shutdown? From what I can tell it is.
+	latestHead, err := oc.ctrl.StopSequencer(oc.shutdownCtx)
+	if err == nil {
+		// None of the consensus state should have changed here so don't log it again.
+		oc.log.Info("stopped sequencer", "latestHead", latestHead)
+	} else {
+		if errors.Is(err, driver.ErrSequencerAlreadyStopped) {
```

Comment:
I'm not sure, but maybe the reason `strings.Contains` was used instead of `errors.Is` is because the error is reconstructed from a json rpc response, so it wouldn't have the actual error type. This should be double checked. If it is indeed of the correct error type, then your fix is better!

Reply:
You're right. I was staring at StopSequencer in op-node that I forgot about the RPC. I'll revert this part.